src/reducers/books.js

```diff
@@ -0,0 +1,25 @@
+import { CREATE_BOOK, REMOVE_BOOK } from '../actions/types';
+
+const initialState = [
+  { id: 'B34KS7K1T1T', title: 'Into the Wild', category: 'Biography' },
```

Comment:
### [Optional]
Let's simplify the code.
Kindly be aware that you can use uuidv4 to automatically generate ids. read more 👉 [npm uuid](https://www.npmjs.com/package/uuidv4)

Example:
```
import uuid from 'uuidv4'
id: uuid(): {title: 'Into the Wild', category: 'Biography'}
```